lua/starfall/libs_sh/builtins.lua

```diff
@@ -8,6 +8,8 @@ local userdataLimit, printBurst
 if SERVER then
 	userdataLimit = CreateConVar("sf_userdata_max", "1048576", { FCVAR_ARCHIVE }, "The maximum size of userdata (in bytes) that can be stored on a Starfall chip (saved in duplications).")
 	printBurst = SF.BurstObject("print", "print", 3000, 10000, "The print burst regen rate in Bytes/sec.", "The print burst limit in Bytes")
+else
+	SF.Permissions.registerPrivilege("enablehud", "Allow enabling hud", "Allows the starfall to enable hud rendering", { client = {} })
```

Comment:
`{ client = { default = 1 } }`
The default for the permission should be owner-only, not anyone.